packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -510,6 +514,10 @@ sdk.server('https://eu.api.example.com/v14');`),
       tsup: tsupOptions as JsonObject,
     };
 
+    if (!this.spdxLicense) {
+      delete pkg.license;
```

Comment:
Doing this instead of adding `license` to `pkg` here because I like having `license` slotted into the object where it is instead of *after* `tsup`. 💅